deploy/examples/nfs.yaml

```diff
@@ -55,17 +55,3 @@ spec:
     #priorityClassName:
     # The logging levels: NIV_NULL | NIV_FATAL | NIV_MAJ | NIV_CRIT | NIV_WARN | NIV_EVENT | NIV_INFO | NIV_DEBUG | NIV_MID_DEBUG |NIV_FULL_DEBUG |NB_LOG_LEVEL
     logLevel: NIV_INFO
----
-apiVersion: ceph.rook.io/v1
```

Comment:
What if we keep this example, but comment it out? Or maybe put it as a separate yaml similar to pool-builtin-mgr.yaml?

Reply:
There is already an example of this in `nfs-test.yaml`. Maybe We can comment about overriding it here and point people to that example.